vulnfetcher/ossvulnfetcher/osindexfetcher_test.go

```diff
@@ -1,7 +1,89 @@
 package ossvulnfetcher
 
-import "testing"
+import (
+	"fmt"
+	"log"
+	"testing"
 
-func TestName(t *testing.T) {
-	// t.Error("Not implemented")
+	"github.com/google/go-cmp/cmp"
+	"github.com/nearform/gammaray/pathrunner"
+	"github.com/spacemeshos/go-spacemesh/assert"
```

Comment:
this is probably coming from me as I am a collaborator of go-spacemesh but I think we should not be using assert from that project as it is unstable and assertions library might change. Testify assertions or plain assertions are a better option. Not a big deal for now (till it breaks, then we can change it).

Reply:
:white_check_mark: golang docs I've read don't advise using assertions libs, and advise to use simple code with ifs instead, and if useful, `cmp` to have nice diffs in the output

Reply:
meh... I am a big fan of assertions. Big golang projects use assertions (standard lib mainly) but we can revisit this in the future.